tests/summarizer/test_openai_summarizer.py

```
@@ -0,0 +1,315 @@
+        assert len(description) > 0
+        assert isinstance(topics, list)
+
+        # Verify API was called
+        mock_client.chat.completions.create.assert_called_once()
+
+        # Verify cost tracking
+        mock_cost_tracker.track_openai_usage.assert_called_once()
+
+    @patch('pod_tenuki.summarizer.openai_summarizer.OpenAI')
+    @patch('pod_tenuki.summarizer.openai_summarizer.cost_tracker')
+    def test_generate_summary_truncates_long_text(self, mock_cost_tracker, mock_openai_class, mock_openai_response):
```

Ruff (0.14.5)

Unused method argument: `mock_cost_tracker`

(ARG002)